MISRA.md

```diff
@@ -1,23 +1,22 @@
 # MISRA Compliance
 
-The AWS SigV4 Library files conform to the
-[MISRA C:2012](https://www.misra.org.uk)
+The AWS SigV4 Library files conform to the [MISRA C:2012](https://www.misra.org.uk)
 guidelines, with some noted exceptions. Compliance is checked with Coverity static analysis.
-Deviations from the MISRA standard are listed below:
+The specific deviations, suppressed inline, are listed below.
 
-### Ignored by [Coverity Configuration](tools/coverity/misra.config)
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Directive 4.9 | Advisory | Allow inclusion of function like macros. Asserts and logging macros use function like macros. |
-| Rule 2.4 | Advisory | Allow unused tags. Some compilers warn if types are not tagged. |
-| Rule 2.5 | Advisory | Allow unused macros. |
-| Rule 3.1 | Required | Allow nested comments. C++ style `//` comments are used in example code within Doxygen documentation blocks. |
-| Rule 11.5 | Advisory | Allow casts from `void *`. Functions with `void *` parameters are used while sorting. |
-
-### Flagged by Coverity
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Rule 8.7 | Advisory | API functions are not used by the library outside of the files they are defined; however, they must be externally visible in order to be used by an application. |
+Additionally, [MISRA configuration file](https://github.com/aws/SigV4-for-AWS-IoT-embedded-sdk/blob/main/tools/coverity/misra.config) contains the project wide deviations.
 
 ### Suppressed with Coverity Comments
-*None.*
+To find the violation references in the source files run grep on the source code
```

Comment:
```suggestion
To find the deviation references in the source files run grep on the source code
```

Reply:
Put this change in, is this something we want to open a set of PR's for in the other libraries or just leave it as is for now?

Reply:
I think we should have caught this earlier. For consistency, we should stick to either violation or deviation. But, I think it is not a blocker.